sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/async_client_base.py

```diff
@@ -75,18 +68,16 @@ def __init__(
 
         self._vault_endpoint = vault_endpoint.strip(" /")
 
-        client = kwargs.pop("generated_client", None)
+        client = kwargs.get("generated_client")
         if client:
             # caller provided a configured client -> nothing left to initialize
             self._client = client
             return
 
-        if api_version is None:
-            api_version = KeyVaultClient.DEFAULT_API_VERSION
-
-        config = self._create_config(credential, api_version=api_version, **kwargs)
+        config = self._create_config(credential, **kwargs)
+        transport = kwargs.pop("transport", None)
         pipeline = kwargs.pop("pipeline", None) or self._build_pipeline(config, transport=transport, **kwargs)
-        self._client = KeyVaultClient(credential, api_version=api_version, pipeline=pipeline, aio=True)
+        self._client = KeyVaultClient(credential, pipeline=pipeline, aio=True)
```

Comment:
Was this change intentional?

Reply:
This is half the intended change, need to pass kwargs along as well. Thanks for catching it. I opened #7778 to track testing API versions.